torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -3061,6 +3061,7 @@ def call_function(
             for node in body_gmod.graph.nodes
             if node.op == "placeholder"
         ]
+
```

Comment:
Could we get a summary of what the pending_unbacked problem is?
Also, does the torch.compile version of invoke_subgraph have this issue too?

Reply:
Updated the summary! And yes torch.compile also runs into the same issue if capture_scalar_outputs is set to True.

Reply:
If you're fixing this for compile too, can you add a compile test too?